bin/classify_SingleR.R

```diff
@@ -100,21 +106,42 @@ singler_results <- SingleR::classifySingleR(
   BPPARAM = bp_param
 )
 
-# add reference name as metadata in singler_results DataFrame
+# add reference name to singler_results DataFrame metadata
 metadata(singler_results)$reference_name <- reference_name
 
 
-# export results
-
-# first, a stand-alone tsv of annotations with both pruned and full labels
-readr::write_tsv(
-  tibble::tibble(
-    barcode = rownames(singler_results),
-    pruned_labels = singler_results$pruned_labels
-  ),
-  opt$output_singler_annotations_file
+# create data frame of annotations
+annotations_df <- tibble::tibble(
+  barcode = rownames(singler_results),
+  singler_celltype_annotation = singler_results$labels,
 )
 
+# map ontology labels to cell type names, as needed
+if (opt$singler_label_name == "label.ont") {
```

Comment:
If you take my previous suggestion
```suggestion
if (opt$singler_uses_ontology) {
```

Reply:
Better suggestion!
```suggestion
if ("cell_ontology_df" %in% names(singler_model)) {
```

Reply:
noting unneeded since https://github.com/AlexsLemonade/scpca-nf/pull/490/commits/a854b7ae21030350aff8c865f882d2e910952af0